docs/index.md

```diff
@@ -78,7 +78,7 @@ Add-Content -Path $PROFILE -Value '(& pixi completion --shell powershell) | Out-
 ### Fish
 
 ```fish
-echo 'pixi completion --shell fish | source' >> ~/.config/fish/config.fish
+pixi completion --shell > ~/.config/fish/completions/pixi.fish
```

Comment:
```suggestion
echo 'pixi completion --shell | source' >> ~/.config/fish/completions/pixi.fish
```
This seems to work and completions should only be sourced once when requested. So it would clean up the `fish.config` and load when needed.